src/components/Settings/Settings.scss

```
@@ -787,6 +787,204 @@
+// Soniox voice-cloning confirm modal (SonioxCloneConfirmModal): playback +
+// naming + the consent statement folded into the accept button. Mirrors
+// .license-consent-modal's layout above. This modal renders inside Modal's
+// .modal-content, OUTSIDE .settings-section/.config-section — the canonical
+// .text-input rule and the shared @keyframes spin below are both scoped
+// (nested selector / same-file-only convention) in ways that don't reach in
+// here, so this block is deliberately self-contained: it duplicates the
+// input treatment and defines its own spin keyframes rather than reaching
+// for either.
+.soniox-clone-confirm-modal {
+  text-align: left;
+  // .modal-content defines no font-size, so text here would otherwise
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)